sentry/src/db/campaign.rs

```diff
@@ -72,7 +121,12 @@ mod test {
 
         assert!(is_inserted);
```

Comment:
Can you please add test against already existing Campaign? same as `is_inserted` but should error & matching the correct error: `PoolError::Backend(error) if error.code() = SqlState::UNIQUE_VIOLATION`

- [ ] Add test for `Unique Violation` error